components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -116,8 +118,17 @@ func (b *Beb) SyncSubscription(subscription *eventingv1alpha1.Subscription, clea
 		// check if EMS subscription is the same as in the past
 		bebSubscription, err = b.getSubscription(sEv2.Name)
 		if err != nil {
-			log.Errorw("get BEB subscription failed", "error", err)
-			return false, err
+			log.Errorw("get BEB subscription failed", "bebSubscriptionName", sEv2.Name, "error", err)
```

Comment:
bebSubscriptionName should also be a constant
same for error
```suggestion
			log.Errorw("get BEB subscription failed", BEB_SUBSCRIPTION_NAME, sEv2.Name, ERROR, err)
```

Reply:
True, but we have another issues where we should define these constants for the complete eventing-controller and not only for this particular file. Changing it later on a global level will be a better solution.

Reply:
Changed to use constants for log keys. Later we will refactor this to define global log keys for all eventing-controller packages, to allow running queries on logs